AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/allocate-hearings/allocate-hearings.component.scss

```diff
@@ -17,6 +17,9 @@
 .no-border {
   border-bottom: none;
 }
+#clockIcon {
```

Comment:
There's not need for a float here. If you add a new column this will make sure the icons are aligned correctly